src/host/getset.cpp

```diff
@@ -1356,24 +1356,21 @@ void DoSrvPrivateAllowCursorBlinking(SCREEN_INFORMATION& screenInfo, const bool
         if (screenInfo.IsCursorInMargins(oldCursorPosition))
         {
             // Cursor is at the top of the viewport
-            const COORD bufferSize = screenInfo.GetBufferSize().Dimensions();
             // Rectangle to cut out of the existing buffer
             SMALL_RECT srScroll;
             srScroll.Left = 0;
-            srScroll.Right = bufferSize.X;
+            srScroll.Right = SHORT_MAX;
             srScroll.Top = viewport.Top;
-            srScroll.Bottom = viewport.Bottom - 1;
+            srScroll.Bottom = viewport.Bottom;

```

Comment:
Please leave comment on inclusive/exclusiveness of the rectangles here so we understand why this -1 mistake was made and how to not make it again when someone revisits this in the future.

Reply:
This particular case wasn't really a mistake, but it was unnecessary, and forced the need for a separate clip rect. However, I have now documented the inclusiveness of the scroll rects everywhere, and also commented the place that did actually require a conversion from exclusive to inclusive.